types/foundry-vtt-api/types/apps/hud/hud.d.ts

```diff
@@ -0,0 +1,29 @@
+// @TODO:
+
+/**
+ * An abstract base class for displaying a heads-up-display interface bound to a Placeable Object on the canvas
+ */
+declare class BasePlaceableHUD extends Application {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/hud/hud.d.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.